src/main/java/beans/RepeatRequestBean.java

```diff
@@ -9,22 +9,31 @@
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class RepeatRequestBean extends SessionBean {
+    private String ix;
```

Comment:
is using `ix` part of the fix here? i like `formIndex` better -- more understandable